tests/api/test_attributes.py

```diff
@@ -93,13 +92,171 @@ def test_attributes_query(user_api_client, product):
             }
         }
     }
-    """
+"""
+
+
+def test_attributes_query(user_api_client, product):
+    attributes = Attribute.objects
+    query = QUERY_ATTRIBUTES
     response = user_api_client.post_graphql(query)
     content = get_graphql_content(response)
     attributes_data = content["data"]["attributes"]["edges"]
     assert len(attributes_data) == attributes.count()
 
 
+def test_attributes_query_hidden_attribute(user_api_client, product, color_attribute):
+    query = QUERY_ATTRIBUTES
+
+    # hide the attribute
+    color_attribute.visible_in_storefront = False
+    color_attribute.save(update_fields=["visible_in_storefront"])
+
+    attribute_count = Attribute.objects.get_visible_to_user(
+        user_api_client.user
+    ).count()
+    assert attribute_count == Attribute.objects.all().count() - 1
```

Comment:
Why ` Attribute.objects.all().count() - 1`?  How about  `Attribute.objects.filter(visible_in_storefront=True).count()`

Reply:
Because there is only one attribute that should be hidden, so it just seems better to hard-code the one and be sure that we have what we expect (where in backend it does the proper filter).

Reply:
So if you always have only one attribute, you don't need to do `Attribute.objects.all().count() - 1` or `Attribute.objects.filter(visible_in_storefront=True).count()`

```
attribute_count = Attribute.objects.get_visible_to_user(
         user_api_client.user
).count()
assert attribute_count == 1
```